setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name="yk_utils",
-    version="1.1.4",
+    version="1.1.5",
```

Comment:
You should increment minor version because it is a new feature :)

Reply:
I didn't notice this comment before merging. I cancelled this PR flow and updated the version here https://github.com/dev-yoonik/yk-utils-python/pull/18.